src/Sparse/Rref.jl

```diff
@@ -208,4 +208,5 @@ function kernel(M::SMat{T}; side::Symbol = :left) where T <: FieldElement
   elseif side == :left
     return _left_kernel(M)[2]
   end
+  error("unreachable")
```

Comment:
```suggestion
  end
  return _left_kernel(M)[2]
```
Could we do this? Then there is less danger that at some point somebody will remove the unreachable error.

Reply:
This is how `solve` and friends handle this. They only have one of the cases in an if-clause, and the other below.